kernel/src/loader/mod.rs

```diff
@@ -17,6 +17,8 @@ use super::cmdline::Error as CmdlineError;
 use memory_model::{Address, GuestAddress, GuestMemory};
 use sys_util;
 
+const PAGE_SIZE: usize = 4096;
```

Comment:
My understanding is that you are trying to round up to the smallest page size possible, right? If that is so, why not add a comment above, something like:
```
// Smallest page size for the supported architectures.
const PAGE_SIZE: usize = 4096;
```

Reply:
Correct, I'll add that comment